sections/body.tex

```diff
@@ -143,7 +143,10 @@ \subsubsection{Benefits of the cloud-computing approach}
 before interacting with the course materials. They only need a web browser in
 order to run Jupyter notebooks. In addition, this ensures that all students
 start with a ``clean'' computing environment - they had access only to the data,
-scripts, and packages that were required for the class.
+scripts, and packages that were required for the class. One of the workshop
```

Comment:
Sorry - that was too many back-ticks... 

The opening quote should be two back-ticks and the closing quote should be two single apostrophes. Hope that's clearer.